Connected to redis://:**@192.168.1.199:6379/5
mingle: searching for neighbors
mingle: all alone
/home/django/xs_venv/lib64/python3.6/site-packages/celery/fixups/django.py:202: UserWarning: Using settings.DEBUG leads to a memory leak, never use this setting in production environments!
  warnings.warn('Using settings.DEBUG leads to a memory leak, never '
celery@xxscan ready.
Events of group {task} enabled by remote.
Received task: scan.tasks.run[4d192c53-c910-46c7-a6ef-3339048e6cde]  
Task scan.tasks.run[4d192c53-c910-46c7-a6ef-3339048e6cde] succeeded in 2.0929233340000337s: 'Tcp Fast Scan'
Received task: scan.tasks.nmap_tcp_scan[c20d1998-b679-47b7-ae18-b4c2471bd6c8]  
Received task: scan.tasks.nmap_result_import[173bf24e-b592-4485-8ca6-6f4f8227ff5b]  
Task scan.tasks.nmap_tcp_scan[c20d1998-b679-47b7-ae18-b4c2471bd6c8] succeeded in 11.590797231000124s: ['/home/django/xx-scan/datas/nmap_scan/result_ed0807b5-c495-42d7-b00c-0b33991b58cc.xml', UUID('27c78918-344d-4300-9ace-79e1de7c335b'), UUID('a15e697e-4064-4091-b81c-783a2d097d78')]
Task scan.tasks.nmap_result_import[173bf24e-b592-4485-8ca6-6f4f8227ff5b] succeeded in 0.13677055600010135s: [UUID('27c78918-344d-4300-9ace-79e1de7c335b'), UUID('a15e697e-4064-4091-b81c-783a2d097d78')]
Received task: scan.tasks.recodes_and_run[d1e839e3-78d7-4bd5-b281-49667af8343b]  
Task scan.tasks.recodes_and_run[d1e839e3-78d7-4bd5-b281-49667af8343b] succeeded in 0.01616454000009071s: {'scripts_num': 0, 'scripts': [], 'stat': True, 'reason': 'All OK of Tasks Scanner.', 'workspaceid': UUID('27c78918-344d-4300-9ace-79e1de7c335b')}
consumer: Connection to broker lost. Trying to re-establish the connection...
Traceback (most recent call last):
  File "/home/django/xs_venv/lib64/python3.6/site-packages/redis/connection.py", line 177, in _read_from_socket
    raise socket.error(SERVER_CLOSED_CONNECTION_ERROR)
OSError: Connection closed by server.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/django/xs_venv/lib64/python3.6/site-packages/celery/worker/consumer/consumer.py", line 320, in start
    blueprint.start(self)
  File "/home/django/xs_venv/lib64/python3.6/site-packages/celery/bootsteps.py", line 119, in start
    step.start(parent)
  File "/home/django/xs_venv/lib64/python3.6/site-packages/celery/worker/consumer/consumer.py", line 596, in start
    c.loop(*c.loop_args())
  File "/home/django/xs_venv/lib64/python3.6/site-packages/celery/worker/loops.py", line 88, in asynloop
    next(loop)
  File "/home/django/xs_venv/lib64/python3.6/site-packages/kombu/async/hub.py", line 345, in create_loop
    cb(*cbargs)
  File "/home/django/xs_venv/lib64/python3.6/site-packages/kombu/transport/redis.py", line 1039, in on_readable
    self.cycle.on_readable(fileno)
  File "/home/django/xs_venv/lib64/python3.6/site-packages/kombu/transport/redis.py", line 337, in on_readable
    chan.handlers[type]()
  File "/home/django/xs_venv/lib64/python3.6/site-packages/kombu/transport/redis.py", line 714, in _brpop_read
    **options)
  File "/home/django/xs_venv/lib64/python3.6/site-packages/redis/client.py", line 680, in parse_response
    response = connection.read_response()
  File "/home/django/xs_venv/lib64/python3.6/site-packages/redis/connection.py", line 624, in read_response
    response = self._parser.read_response()
  File "/home/django/xs_venv/lib64/python3.6/site-packages/redis/connection.py", line 284, in read_response
    response = self._buffer.readline()
  File "/home/django/xs_venv/lib64/python3.6/site-packages/redis/connection.py", line 216, in readline
    self._read_from_socket()
  File "/home/django/xs_venv/lib64/python3.6/site-packages/redis/connection.py", line 191, in _read_from_socket
    (e.args,))
redis.exceptions.ConnectionError: Error while reading from socket: ('Connection closed by server.',)
consumer: Cannot connect to redis://:**@192.168.1.199:6379/5: Error 111 connecting to 192.168.1.199:6379. Connection refused..
Trying again in 2.00 seconds...

Connected to redis://:**@192.168.1.199:6379/5
mingle: searching for neighbors
mingle: all alone
Connected to redis://:**@192.168.1.199:6379/5
mingle: searching for neighbors
mingle: all alone
/home/django/xs_venv/lib64/python3.6/site-packages/celery/fixups/django.py:202: UserWarning: Using settings.DEBUG leads to a memory leak, never use this setting in production environments!
  warnings.warn('Using settings.DEBUG leads to a memory leak, never '
celery@xxscan ready.
Events of group {task} enabled by remote.
Received task: scan.tasks.run[bac03d2c-5e19-4b3a-b6d9-432331350b6e]  
Task scan.tasks.run[bac03d2c-5e19-4b3a-b6d9-432331350b6e] succeeded in 2.105440112999986s: 'Tcp Fast Scan'
Received task: scan.tasks.nmap_tcp_scan[e55a7c07-58ab-4c18-bf07-89f6d3e3e36b]  
Received task: scan.tasks.nmap_result_import[f4f5f4ca-0988-423a-a5ba-1507f5e9d789]  
Task scan.tasks.nmap_tcp_scan[e55a7c07-58ab-4c18-bf07-89f6d3e3e36b] succeeded in 10.832448070999817s: ['/home/django/xx-scan/datas/nmap_scan/result_7ddca2b0-4299-4a20-a0ee-e48451672204.xml', UUID('27c78918-344d-4300-9ace-79e1de7c335b'), UUID('a15e697e-4064-4091-b81c-783a2d097d78')]
Task scan.tasks.nmap_result_import[f4f5f4ca-0988-423a-a5ba-1507f5e9d789] succeeded in 0.034889868000163915s: [UUID('27c78918-344d-4300-9ace-79e1de7c335b'), UUID('a15e697e-4064-4091-b81c-783a2d097d78')]
Received task: scan.tasks.recodes_and_run[e777ddea-4d17-40d7-8a9a-a8cd4ba45f30]  
Task scan.tasks.recodes_and_run[e777ddea-4d17-40d7-8a9a-a8cd4ba45f30] succeeded in 0.018653081000138627s: {'scripts_num': 0, 'scripts': [], 'stat': True, 'reason': 'All OK of Tasks Scanner.', 'workspaceid': UUID('27c78918-344d-4300-9ace-79e1de7c335b')}
Received task: scan.tasks.run[5b338731-ce41-4cdc-b6e7-fd79294c818d]  
Received task: scan.tasks.nmap_tcp_scan[2a9c60b0-8aa2-47a6-8b90-5887e3733493]  
Task scan.tasks.run[5b338731-ce41-4cdc-b6e7-fd79294c818d] succeeded in 2.031034325000064s: 'Tcp Fast Scan'
Received task: scan.tasks.nmap_result_import[ec4898a0-1995-482e-b7fb-09a415c2b2c6]  
Task scan.tasks.nmap_tcp_scan[2a9c60b0-8aa2-47a6-8b90-5887e3733493] succeeded in 21.49893916200017s: ['/home/django/xx-scan/datas/nmap_scan/result_7ddca2b0-4299-4a20-a0ee-e48451672204.xml', UUID('27c78918-344d-4300-9ace-79e1de7c335b'), UUID('a15e697e-4064-4091-b81c-783a2d097d78')]
Received task: scan.tasks.recodes_and_run[de18998c-f963-4bcf-81bc-ee32fe4bf849]  
Task scan.tasks.nmap_result_import[ec4898a0-1995-482e-b7fb-09a415c2b2c6] succeeded in 0.05159594700012349s: [UUID('27c78918-344d-4300-9ace-79e1de7c335b'), UUID('a15e697e-4064-4091-b81c-783a2d097d78')]
redis没有注册服务
Received task: scan.tasks.push_cmd[cd89a4f4-7ec2-4e2a-a3aa-da7a13baec59]  
Received task: scan.tasks.push_cmd[f8df637f-420f-42fc-8884-0e5b01e639a1]  
Received task: scan.tasks.push_cmd[eaeae788-2fa0-4d15-8ec6-f51c40b52089]  
Received task: scan.tasks.push_cmd[8731281c-d984-4419-bb90-3f5a0f5158d6]  
Scaling up 1 processes.
Task scan.tasks.recodes_and_run[de18998c-f963-4bcf-81bc-ee32fe4bf849] succeeded in 0.7395719030000691s: {'scripts_num': 9, 'scripts': ['nmap -sC -sV -Pn -v -p 8888 --script=http-vuln* --script-args http-vuln-cve2014-3704.cmd="uname -a",http-vuln-cve2014-3704.uri="/drupal" 127.0.0.1 -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_http_vuln-127.0.0.1-8888.txt --host-timeout 40m --script-timeout 20m', 'nmap -sC -sV -Pn -v -p 8888 --script=http-* 127.0.0.1 -d -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_script_http-127.0.0.1-8888.txt -oX /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_script_http-127.0.0.1-8888.xml --host-timeout 120m --script-timeout 20m', 'nmap 127.0.0.1 -p 8888 -Pn --script='http* and not (broadcast or dos or external or http-slowloris* or fuzzer)' -oN [output].txt --host-timeout 40m --script-timeout 20m', 'nmap 127.0.0.1 -p 22 -sC -sV -v -Pn -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_service_scan-127.0.0.1-22.txt', 'nmap -sC -sV -v -Pn 127.0.0.1 -p 111 -oN...', 'nmap -sC -sV -Pn -v -p 5555...', 'nmap -sC...', ...]}
Received task: scan.tasks.push_cmd[534eb5e2-7fee-4f04-90c1-e220cab6fd94]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[e2db0f8b-0218-4151-8341-f8b532c1e381]  
Received task: scan.tasks.push_cmd[2ef725b4-696c-4c85-8a24-150277103901]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[250eaf88-fbbd-43f3-b7f5-cb9390c109e2]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[9a9248c6-8d88-4382-9025-c3d8181a1926]  
Scaling up 1 processes.
Task scan.tasks.push_cmd[cd89a4f4-7ec2-4e2a-a3aa-da7a13baec59] succeeded in 1.023360324999885s: {'stat': True, 'cmd': 'nmap -sC -sV -Pn -v -p 8888 --script=http-vuln* --script-args http-vuln-cve2014-3704.cmd="uname -a",http-vuln-cve2014-3704.uri="/drupal" 127.0.0.1 -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_http_vuln-127.0.0.1-8888.txt --host-timeout 40m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[f8df637f-420f-42fc-8884-0e5b01e639a1] succeeded in 1.0323259859999325s: {'stat': True, 'cmd': 'nmap -sC -sV -Pn -v -p 8888 --script=http-* 127.0.0.1 -d -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_script_http-127.0.0.1-8888.txt -oX /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_script_http-127.0.0.1-8888.xml --host-timeout 120m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[eaeae788-2fa0-4d15-8ec6-f51c40b52089] succeeded in 1.0270641870001782s: {'stat': True, 'cmd': 'nmap 127.0.0.1 -p 8888 -Pn --script='http* and not (broadcast or dos or external or http-slowloris* or fuzzer)' -oN [output].txt --host-timeout 40m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[e2db0f8b-0218-4151-8341-f8b532c1e381] succeeded in 1.1916130709998924s: {'stat': True, 'cmd': 'nmap -sC -sV -Pn -v -p 5555 --script=http-vuln* --script-args http-vuln-cve2014-3704.cmd="uname -a",http-vuln-cve2014-3704.uri="/drupal" 127.0.0.1 -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_http_vuln-127.0.0.1-5555.txt --host-timeout 40m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[250eaf88-fbbd-43f3-b7f5-cb9390c109e2] succeeded in 1.0309522540001126s: {'stat': True, 'cmd': 'nmap 127.0.0.1 -p 5555 -Pn --script='http* and not (broadcast or dos or external or http-slowloris* or fuzzer)' -oN [output].txt --host-timeout 40m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[2ef725b4-696c-4c85-8a24-150277103901] succeeded in 1.048732530000052s: {'stat': True, 'cmd': 'nmap -sC -sV -Pn -v -p 5555 --script=http-* 127.0.0.1 -d -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_script_http-127.0.0.1-5555.txt -oX /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_script_http-127.0.0.1-5555.xml --host-timeout 120m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[9a9248c6-8d88-4382-9025-c3d8181a1926] succeeded in 1.0570158150001134s: {'stat': True, 'cmd': 'nmap 127.0.0.1 -p 25 -sC -sV -v -Pn  -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_service_scan-127.0.0.1-25.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[8731281c-d984-4419-bb90-3f5a0f5158d6] succeeded in 1.9012107730000025s: {'stat': True, 'cmd': 'nmap 127.0.0.1 -p 22 -sC -sV -v -Pn -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_service_scan-127.0.0.1-22.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[534eb5e2-7fee-4f04-90c1-e220cab6fd94] succeeded in 7.543435666999812s: {'stat': True, 'cmd': 'nmap -sC -sV -v -Pn 127.0.0.1 -p 111 -oN /home/django/xx-scan/results/admin/xadmin/127.0.0.1/20190611/nmap_service_scan-127.0.0.1-111.txt', 'reason': '远程执行命令执行成功！'}
Scaling down 5 processes.
Received task: scan.tasks.run[70b07b08-2915-4aa6-82cb-8cf2a3ef9bf4]  
Task scan.tasks.run[70b07b08-2915-4aa6-82cb-8cf2a3ef9bf4] succeeded in 2.1211557880001237s: 'Tcp Fast Scan'
Received task: scan.tasks.nmap_tcp_scan[8f92fd21-88af-429d-9431-24ea49ff259d]  
Task scan.tasks.nmap_tcp_scan[8f92fd21-88af-429d-9431-24ea49ff259d] succeeded in 115.93682843399984s: ['/home/django/xx-scan/datas/nmap_scan/result_5797b2ac-5441-4878-9971-a1c7b77b3ae9.xml', UUID('41436157-8498-447a-a2c2-97c0fcfce5e5'), UUID('20f73e27-0e0f-4db8-9e7b-a4a3f6659999')]
Received task: scan.tasks.nmap_result_import[4fa1ed12-ff9e-4b73-9321-e5d5f910ba67]  
Received task: scan.tasks.recodes_and_run[05d78028-e93b-4ffa-8087-6ae45401e386]  
Task scan.tasks.nmap_result_import[4fa1ed12-ff9e-4b73-9321-e5d5f910ba67] succeeded in 0.07652931000029639s: [UUID('41436157-8498-447a-a2c2-97c0fcfce5e5'), UUID('20f73e27-0e0f-4db8-9e7b-a4a3f6659999')]
postgresql没有注册服务
Received task: scan.tasks.push_cmd[b6c628bd-0fd6-46c5-8ddb-415c00d38171]  
Received task: scan.tasks.push_cmd[967630ce-1019-43d5-a678-e43c049e7e53]  
Received task: scan.tasks.push_cmd[7d9ce515-461d-4a33-81f6-e539c42355cb]  
Received task: scan.tasks.push_cmd[23cf075e-837c-4dd6-8ac8-b977e23468e2]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[436e9bb2-c35b-4bc4-a7bf-00f304c4dbc6]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[3dd9061b-9423-432d-9581-25b9b551b866]  
Scaling up 1 processes.
Task scan.tasks.recodes_and_run[05d78028-e93b-4ffa-8087-6ae45401e386] succeeded in 2.063007650000145s: {'scripts_num': 24, 'scripts': ['nikto -h http://192.168.1.10:80//robot.txt -output /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nikto-192.168.1.10-80.txt', 'gobuster -u http://192.168.1.10:80//robot.txt -f -k -w /usr/share/seclists/Discovery/Web-Content/common.txt -s '200,204,302,307,403,500' -e -n -q > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/gobuster_common-192.168.1.10-80.txt', 'nmap -sC -sV -Pn -v -p 80 --script=http-vuln* --script-args http-vuln-cve2014-3704.cmd="uname -a",http-vuln-cve2014-3704.uri="/drupal" 192.168.1.10 -oN /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nmap_http_vuln-192.168.1.10-80.txt --host-timeout 40m --script-timeout 20m', 'sqlmap -u http://192.168.1.10:80//robot.txt --batch --crawl=5 --level 1 --risk 1 -f -a --smart -v0 --disable-coloring > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/sqlmap-192.168.1.10-80.txt', 'cewl http://192.168.1.10:80//robot.txt -m 6 -w...', 'wpscan --url...', 'curl...', ...]}
Received task: scan.tasks.push_cmd[397631fe-60d9-485e-aebb-d3a788b2b651]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[f24aa3f4-a8a0-4d46-9533-6dc49fcd1b7b]  
Received task: scan.tasks.push_cmd[66210d53-b896-4a2f-89b5-a350c9819359]  
Scaling up 1 processes.
Task scan.tasks.push_cmd[b6c628bd-0fd6-46c5-8ddb-415c00d38171] succeeded in 1.0132975800001986s: {'stat': True, 'cmd': 'nikto -h http://192.168.1.10:80//robot.txt -output /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nikto-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Received task: scan.tasks.push_cmd[b3fe66ba-d50a-4ed0-b180-f2f72e078641]  
Scaling up 1 processes.
Task scan.tasks.push_cmd[967630ce-1019-43d5-a678-e43c049e7e53] succeeded in 1.016775872999915s: {'stat': True, 'cmd': 'gobuster -u http://192.168.1.10:80//robot.txt -f -k -w /usr/share/seclists/Discovery/Web-Content/common.txt -s '200,204,302,307,403,500' -e -n -q > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/gobuster_common-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Received task: scan.tasks.push_cmd[9047b712-6630-47a7-a49b-8b7814c7446d]  
Task scan.tasks.push_cmd[7d9ce515-461d-4a33-81f6-e539c42355cb] succeeded in 1.112142522999875s: {'stat': True, 'cmd': 'nmap -sC -sV -Pn -v -p 80 --script=http-vuln* --script-args http-vuln-cve2014-3704.cmd="uname -a",http-vuln-cve2014-3704.uri="/drupal" 192.168.1.10 -oN /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nmap_http_vuln-192.168.1.10-80.txt --host-timeout 40m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Received task: scan.tasks.push_cmd[4ae5f231-47fe-4eef-892c-8e2eeaaac771]  
Received task: scan.tasks.push_cmd[245f09a4-2616-48e6-bf16-93a1c469cea5]  
Received task: scan.tasks.push_cmd[59048eb2-3f00-4c81-ad5a-b1bfdc48fbc7]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[eea6cc43-9afc-468e-b276-89c227eed397]  
Scaling up 1 processes.
Received task: scan.tasks.push_cmd[93e1deb7-7b17-4070-8a62-ec52a3621d75]  
Scaling up 1 processes.
Task scan.tasks.push_cmd[23cf075e-837c-4dd6-8ac8-b977e23468e2] succeeded in 1.0137218980003126s: {'stat': True, 'cmd': 'sqlmap -u http://192.168.1.10:80//robot.txt --batch --crawl=5 --level 1 --risk 1 -f -a --smart -v0 --disable-coloring > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/sqlmap-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Received task: scan.tasks.push_cmd[98d3be4a-211d-4dc3-8d10-34b3e3291f3b]  
Scaling up 1 processes.
Task scan.tasks.push_cmd[397631fe-60d9-485e-aebb-d3a788b2b651] succeeded in 1.0165223179997156s: {'stat': True, 'cmd': 'curl http://192.168.1.10:80//robot.txt/robots.txt --user-agent 'Googlebot/2.1 (+http://www.google.com/bot.html)' --connect-timeout 30 --max-time 180  > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/curl_robots-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[436e9bb2-c35b-4bc4-a7bf-00f304c4dbc6] succeeded in 1.0290204249999988s: {'stat': True, 'cmd': 'cewl http://192.168.1.10:80//robot.txt -m 6 -w /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/cewl-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[3dd9061b-9423-432d-9581-25b9b551b866] succeeded in 1.062757068999872s: {'stat': True, 'cmd': 'wpscan --url http://192.168.1.10:80//robot.txt --disable-tls-checks --no-banner -f cli-no-color --enumerate p t tt u > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/wpscan-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Received task: scan.tasks.push_cmd[643f4995-2b91-4811-9eec-9e789cc7cafa]  
Received task: scan.tasks.push_cmd[342bff8a-5684-4094-803e-dbfbda6aad99]  
Received task: scan.tasks.push_cmd[7bdaefa2-a360-4b32-8b4b-6f230dc32543]  
Received task: scan.tasks.push_cmd[8ffe7316-faee-4c99-839a-6fecf224c545]  
Task scan.tasks.push_cmd[f24aa3f4-a8a0-4d46-9533-6dc49fcd1b7b] succeeded in 1.0151080509999701s: {'stat': True, 'cmd': 'gobuster -u http://192.168.1.10:80//robot.txt -f -k -w /usr/share/wordlists/dirbuster/directory-list-lowercase-2.3-medium.txt -s '200,204,307,403,500' -e -n -q > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/gobuster_2.3-medium-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Received task: scan.tasks.push_cmd[bc45d985-38b3-47b0-98c4-7a30eb8d521c]  
Scaling up 1 processes.
Task scan.tasks.push_cmd[9047b712-6630-47a7-a49b-8b7814c7446d] succeeded in 1.0391663929999595s: {'stat': True, 'cmd': 'gobuster -u http://192.168.1.10:80//robot.txt -f -k -w /usr/share/seclists/Discovery/Web_Content/cgis.txt -s '200,204,301,307,403,500' -e -n -q > '/home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/gobuster_cgis-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[66210d53-b896-4a2f-89b5-a350c9819359] succeeded in 1.041477877000034s: {'stat': True, 'cmd': 'nmap -sC -sV -Pn -v -p 80 --script=http-* 192.168.1.10 -d -oN /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nmap_script_http-192.168.1.10-80.txt -oX /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nmap_script_http-192.168.1.10-80.xml --host-timeout 120m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[b3fe66ba-d50a-4ed0-b180-f2f72e078641] succeeded in 1.0500332249998792s: {'stat': True, 'cmd': 'curl -i http:/192.168.1.10:80//robot.txt -o /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/curl_main-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[245f09a4-2616-48e6-bf16-93a1c469cea5] succeeded in 1.0267260649998207s: {'stat': True, 'cmd': 'whatweb http://192.168.1.10:80//robot.txt -a3 --colour=never > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/whatweb-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[4ae5f231-47fe-4eef-892c-8e2eeaaac771] succeeded in 1.0456034139997428s: {'stat': True, 'cmd': 'python /opt/Photon/photon.py -u http://192.168.1.10:80//robot.txt -o /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/photon-192.168.1.10-80 -e json && cat /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/photon-192.168.1.10-80/exported.json > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/photon-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Received task: scan.tasks.push_cmd[a572ed60-2d9a-4d11-83b8-479ad1143598]  
Received task: scan.tasks.push_cmd[6f0e1775-0cd8-430e-a3a5-fb78a4f34987]  
Task scan.tasks.push_cmd[98d3be4a-211d-4dc3-8d10-34b3e3291f3b] succeeded in 1.0568414130002566s: {'stat': True, 'cmd': 'rpcclient -U \"\" 192.168.1.10 -p 111 -N > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/rpcclient-192.168.1.10-111.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[643f4995-2b91-4811-9eec-9e789cc7cafa] succeeded in 1.0518645470001502s: {'stat': True, 'cmd': 'nmblookup -A 192.168.1.10 > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nmblookup-192.168.1.10-111.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[59048eb2-3f00-4c81-ad5a-b1bfdc48fbc7] succeeded in 1.0977926079999634s: {'stat': True, 'cmd': 'gobuster -u http://192.168.1.10:80//robot.txt -f -k -w /usr/share/seclists/Discovery/Web-Content/RobotsDisallowed-Top1000.txt -s '200,204,302,307,403,500' -e -n -q > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/gobuster_robots-dis-192.168.1.10-80.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[eea6cc43-9afc-468e-b276-89c227eed397] succeeded in 1.0995160050001687s: {'stat': True, 'cmd': 'nmap 192.168.1.10 -p 80 -Pn --script='http* and not (broadcast or dos or external or http-slowloris* or fuzzer)' -oN [output].txt --host-timeout 40m --script-timeout 20m', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[342bff8a-5684-4094-803e-dbfbda6aad99] succeeded in 1.0256596029998946s: {'stat': True, 'cmd': 'smbclient -L //192.168.1.10 -p 111 -N > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/smbclient-192.168.1.10-111.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[7bdaefa2-a360-4b32-8b4b-6f230dc32543] succeeded in 1.0290007080002397s: {'stat': True, 'cmd': 'enum4linux -a 192.168.1.10 > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/enum4linux-192.168.1.10-111.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[8ffe7316-faee-4c99-839a-6fecf224c545] succeeded in 1.0758117230002426s: {'stat': True, 'cmd': 'ncrack -vv -p 22 --user root -P /usr/share/seclists/Passwords/Common-Credentials/best15.txt 192.168.1.10 > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/ncrack_ssh_best15-192.168.1.10-22.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[6f0e1775-0cd8-430e-a3a5-fb78a4f34987] succeeded in 1.0599755260000165s: {'stat': True, 'cmd': 'hydra -f -V -t 1 -l root -P /usr/share/seclists/Passwords/Common-Credentials/best15.txt -s 22 192.168.1.10 ssh > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/hydra_ssh_best15-192.168.1.10-22.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[bc45d985-38b3-47b0-98c4-7a30eb8d521c] succeeded in 1.1001037229998474s: {'stat': True, 'cmd': 'medusa -u root -P /usr/share/seclists/Passwords/Common-Credentials/best15.txt -e ns -h 192.168.1.10 - 22 -M ssh > /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/medusa-192.168.1.10-22.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[a572ed60-2d9a-4d11-83b8-479ad1143598] succeeded in 1.650387968999894s: {'stat': True, 'cmd': 'nmap 192.168.1.10 -p 22 -sC -sV -v -Pn -oN /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nmap_service_scan-192.168.1.10-22.txt', 'reason': '远程执行命令执行成功！'}
Task scan.tasks.push_cmd[93e1deb7-7b17-4070-8a62-ec52a3621d75] succeeded in 7.403357643000163s: {'stat': True, 'cmd': 'nmap -sC -sV -v -Pn 192.168.1.10 -p 111 -oN /home/django/xx-scan/results/admin/test2222/192.168.1.10/20190611/nmap_service_scan-192.168.1.10-111.txt', 'reason': '远程执行命令执行成功！'}
Scaling down 11 processes.
